data_structures/stacks/stock_span_problem.py

```diff
@@ -8,44 +8,44 @@
 """
 
 
-def calculation_span(price, s):
+def calculation_span(price):
```

Comment:
```suggestion
def calculation_span(price: list[int]):
```
Or even better `list[float]`

Reply:
I changed it to list[float]